about card metric image tag removing /

Jfong1218 opened the pull request:
Fixes #5253

### What changes did you make?
 - Removed ending slash for img tag of metrics.svg

### Why did you make the changes (we will use this info to test)?
  - Needed a consistent codebase with how we use img HTML tags

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)
<!-- Note, if your images are too big, use the <img src="" width="" length="" />  syntax instead of ![image](link) to format the images -->
<!-- If images are not loading properly, you might need to double check the syntax or add a newline after the closing </summary> tag -->

no visual change

Jfong1218 committed:
about card metric image tag removing /

Jfong1218 committed:
Removing image tag / refractor

meru-a commented:
Availability: In the evening & on weekends
ETA: EOD 08/27

meru-a commented on _includes/about-page/about-card-metrics.html:
```diff
@@ -13,7 +13,7 @@
             <!--This is the top and then right cell-->
             <div class="top-left-column">
-                <div class="small-card">
+                <div class="small-card">w
```
@Jfong1218 Here is the line with an extra character at the end. Please remove the character and re-submit the changes, thank you!

christinaor replied:
The extra character has been resolved in the latest changes.

meru-a reviewed (changes requested):
- The new branch is properly going into the correct branch gh-pages. ✅
- Original issue was properly linked and referenced. ✅
- The specified instance of an `<img>` tag was properly changed. ✅
- No additional changes were made. ❌

Great work! However, on the line 16 of file `_includes/about-page/about-card-metrics.html` there is an extra character 'w' that shouldn't be there. After this small edit, I'll be glad to approve your pull request!

Jfong1218 committed:
removing w from line 16

Jfong1218 commented:
Fixes #5253

### What changes did you make?
Removed ending slash for img tag of metrics.svg
also removed w after line 16 
### Why did you make the changes (we will use this info to test)?
Needed a consistent codebase with how we use img HTML tags
accidentally typed w on line 16
### Screenshots of Proposed Changes Of The Website (if any, please do not screen shot code changes)
No visual changes

adrianang commented:
> Fixes #5253
> 
> ### What changes did you make?
> Removed ending slash for img tag of metrics.svg also removed w after line 16
> 
> ### Why did you make the changes (we will use this info to test)?
> Needed a consistent codebase with how we use img HTML tags accidentally typed w on line 16
> 
> ### Screenshots of Proposed Changes Of The Website (if any, please do not screen shot code changes)
> No visual changes

Hi @Jfong1218 — you'll want to _edit the original post_ of your PR (go to the top-right of your original post in this PR and click 'Edit' in the dropdown) with these updates instead. Also, you don't need to make note of the fix to the typo you made in the PR; the PR should only document whatever will get merged into the HfLA Website codebase (so bullets for just the removal of the ending slash, as the typo wouldn't get merged in).

meru-a reviewed (approved):
The extra character seems to be removed, so I'm approving the request. Great job!

kwangric commented:
Hey @DorianDeptuch! When you get the chance, please leave a comment with an availability and ETA for your review. Thanks!

DorianDeptuch commented:
Availability: 8/28 Afternoon
ETA: 8/28 EOD

Jfong1218 commented:
> > Fixes #5253
> > ### What changes did you make?
> > Removed ending slash for img tag of metrics.svg also removed w after line 16
> > ### Why did you make the changes (we will use this info to test)?
> > Needed a consistent codebase with how we use img HTML tags accidentally typed w on line 16
> > ### Screenshots of Proposed Changes Of The Website (if any, please do not screen shot code changes)
> > No visual changes
> 
> Hi @Jfong1218 — you'll want to _edit the original post_ of your PR (go to the top-right of your original post in this PR and click 'Edit' in the dropdown) with these updates instead. Also, you don't need to make note of the fix to the typo you made in the PR; the PR should only document whatever will get merged into the HfLA Website codebase (so bullets for just the removal of the ending slash, as the typo wouldn't get merged in).

DorianDeptuch reviewed (commented):
Hi @Jfong1218,

I ran this in my local environment and everything seems to work well. You removed the ending img slash correctly. In the discussion, you have the What and the Why sections done correctly and there are no visual changes. I would consider editing your post to remove the `<details>` from the visual changes section and instead writing something along the line of "No visual changes." You can follow @adrianang 's comment in the conversation to find the instructions to edit your post. It is not necessary to mention anything about removing the "w" in the What or the Why sections because it does not pertain to the original issue.

Besides that, everything looks good to me. Great job. Re-request me for a review when you make the changes and I will approve the PR.

Jfong1218 commented:
Hello Team,

I've made the necessary changes in my PR per @DorianDeptuch and @adrianang request and have deleted the duplicate comment.

Thanks!

christinaor commented:
Hi @Jfong1218 , good job on this issue! It looks like you made the necessary changes based on the feedback in comments. I will merge this, but in the future, you may request a Hack for LA member to review your issue again by clicking on the revolving circle next to their username (see the red square in the image):

![image](https://github.com/hackforla/website/assets/61027932/aabe7f04-7b4d-4281-a81f-c3d7a24b1c37)
